snippets/visualbasic/VS_Snippets_CLR_System/system.Globalization.DateTimeFormatInfo.LongDatePattern/VB/longdatepattern1.vb

```diff
@@ -2,9 +2,10 @@
 Option Strict On
 
 ' <Snippet2>
+Imports System
 Imports System.Globalization
 
-Module Example
+Public Class Example
```

Comment:
This code won't compile because it lacks an entry point. Please revert this change.